File: app/src/main/java/com/eftimoff/bakingapp/recipedetails/view/RecipeStepsViewHolder.java
package com.eftimoff.bakingapp.recipedetails.view;

import android.view.View;
import android.widget.TextView;

import com.eftimoff.bakingapp.R;
import com.eftimoff.bakingapp.app.models.Step;
import com.eftimoff.bakingapp.app.view.BaseViewHolder;

import butterknife.BindView;

public class RecipeStepsViewHolder extends BaseViewHolder<Step> {

    @BindView(R.id.recipeStepTitle)
    TextView recipeStepTitle;
    @BindView(R.id.recipeStepNumber)
    TextView recipeStepNumber;

    public RecipeStepsViewHolder(View itemView) {
        super(itemView);
    }

    @Override
    protected void bind(Step step) {
        recipeStepTitle.setText(step.getShortDescription());
        recipeStepNumber.setText(String.valueOf(step.getId() + 1));
    }
}

File: app/src/main/java/com/eftimoff/bakingapp/recipedetails/view/RecipeStepsViewHolder.kt
package com.eftimoff.bakingapp.recipedetails.view

import android.view.View
import com.eftimoff.bakingapp.app.models.Step
import com.eftimoff.bakingapp.app.view.BaseViewHolder
import kotlinx.android.synthetic.main.item_recipe_step.view.*

class RecipeStepsViewHolder(itemView: View, var callback: RecipeStepsAdapter.Callback?) : BaseViewHolder<Step>(itemView) {

    override fun bind(t: Step) {
        itemView.recipeStepTitle.text = t.shortDescription
        itemView.recipeStepNumber.text = (t.id + 1).toString()
    }

}